servicetalk-serialization-api/src/test/java/io/servicetalk/serialization/api/DefaultSerializerDeserializationTest.java

```diff
@@ -302,15 +297,15 @@ public void deserializeIncompleteAggregatedWithType() {
 
         try {
             deserialized.iterator().hasNext();
-            fail();
+            Assertions.fail();
```

Comment:
Try to use static-imports for things within the context that are easy to understand.